unreleased/chloggen.yaml

```diff
@@ -0,0 +1,11 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern
+component: cmd/chloggen
+
+# A single sentence describing the change
+note: Add new tool for conflict-free CHANGELOG.md management
+
+# One or more tracking issues related to the change
+issues: [ 11539 ]
```

Comment:
Don't we normally link to the PR in the changelog?

Reply:
Yes, however, I'm proposing that we switch to using issues. See conversation here:
https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/11540#discussion_r909193195

Reply:
I'm good with that, I hate having to update the CHANGELOG after I open the PR lol